rviz_default_plugins/CMakeLists.txt

```diff
@@ -241,6 +241,7 @@ ament_target_dependencies(rviz_default_plugins
 
 ament_export_include_directories(include)
 ament_export_interfaces(rviz_default_plugins)
+ament_export_libraries(rviz_default_plugins)
```

Comment:
I wonder if this is redundant with `ament_export_interfaces`, or rather whether or not that function should also extend the library paths.

@dirk-thomas what do you think?

Reply:
While both do kind of the same thing they work differently:
* `ament_export_interfaces` is used to export the targets via CMake `install(EXPORT ...)`
*  `ament_export_libraries` exports libraries using the variable `<pkgname>_LIBRARIES`

Both make sense depending on how downstream packages want to use them.

Since the target is a library `ament_export_interfaces` should pass the option `HAS_LIBRARY_TARGET` (see https://github.com/ament/ament_cmake/blob/88add8cab0ca579e89e95b3ab343af319c163195/ament_cmake_export_interfaces/cmake/ament_export_interfaces.cmake#L22-L24).

`HAS_LIBRARY_TARGET`